src/components/Pressable/Pressable.tsx

```diff
@@ -358,11 +358,7 @@ export default function Pressable(props: PressableProps) {
     gesture.enabled(isPressableEnabled);
     gesture.runOnJS(true);
     gesture.hitSlop(appliedHitSlop);
-    gesture.shouldCancelWhenOutside(false);
-
-    if (Platform.OS !== 'web') {
-      gesture.shouldCancelWhenOutside(true);
-    }
+    gesture.shouldCancelWhenOutside(Platform.OS === 'web' ? false : true);
```

Comment:
Can it be like `gesture.shouldCancelWhenOutside(Platform.OS !== 'web');`

Reply:
Yes, it could 😅